search-scores-and-weights.md

```diff
@@ -0,0 +1,40 @@
+Our search system uses scoring to assign values to each document based on
+different parameters. The interplay between clustering and scoring helps
+optimize the search process, ensuring that the system considers both the content
+and context to deliver more accurate and relevant results for the user.
+
+This task becomes challenging when dealing with a large number of documents,
+creating the need for optimization strategies. A good approach is the
+utilization of clustering techniques, which categorize documents based on their
+topics, facilitating a more streamlined and efficient search process. We also
+use scoring, which involves the assignment of numerical values and weights to
+documents based on various parameters, influencing their ranking in the search
+results. Here is our different scores, each serving a unique purpose:
+
+1. **Recency**: This score considers the temporal aspect of documents,
+   prioritizing recently added or updated content. A document's recency is
+   crucial in reflecting the latest information available to users.
+
+1. **Traffic**: The frequency with which users consult a document influences its
+   score. Popular or frequently accessed documents are given higher scores,
+   indicating their relevance and importance to users. Warning: The home page is
+   rated really high since it's where every user land at first.
+
+1. **Current**: This score determines whether a document is currently accessible
+   (current=1) or if it has been archived (current=0). It helps users
+   distinguish between active and inactive content.
+
+1. **Typicality**: This score assesses whether a document is a typical
+   informative representation of the broader concept or topic being queried.
+   Typical documents are considered representative examples within a given
+   context and are assigned higher scores. This ensures that the search results
+   prioritize documents that serve as typical and informative instances within
+   the targeted topic.
+
```

Comment:
@JolanThomassin also provide information from your new score

Reply:
**Didactic:** This score evaluates the informational value within content chunks that lack a defined structure such as tables. It focuses on the quality and depth of information provided without relying on tabular formats for organization or presentation. Documents with high didactic scores often contain rich textual information, explanations, and details without reliance on tabulated data.

**Guidance:** This score pertains to content chunks extracted from guidance-oriented pages, emphasizing their significance and relevance. Guidance pages typically offer comprehensive direction, instruction, or expert advice within a specific domain. As these pages tend to provide crucial information or instructions sought by users, they are given priority to ensure users can readily access the most helpful and directive content.

Reply:
added !